CMakeLists.txt

```diff
@@ -450,6 +452,9 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
         COMMAND ln -sf libopencv_imgcodecs.so libopencv_imgcodecs.so.${OPENCV_VERSION}
```

Comment:
@mc-nv Can you remind me which torch library required these libraries?

Reply:
This lib is required by `libopencv_videoio.so`.